### ISSUE TITLE
Crash observed at client application which uses v1.6.4

### ISSUE THREAD

**Issue by ymalli144** [eclipse/mosquitto#1407]
Hi,

We have compiled a client with version 1.4.3 and now using 1.6.4 DLLs(mosquitto.dll and mosquittopp.dll) in production. A crash is observed while trying to make the connection with MQTT broker. I have done debug of my client application and observed that connect() API is crashing. Can you please help to recover from this crash?

**Comment by ralight** [comment]
Could you provide some example code please? Without more details it will be difficult to help.

**Comment by ymalli144** [comment]
It is not with the code, we have implemented two clients (MQTT_Pub and MQTT_Sub) on Mosqiotto v1.4.3 and working fine. Now we are planning to use Mosquitto v1.6.4 at MQTT_Pub client and no change in MQTT_Sub client. Here both the clients run in same machine and computer is updated with new set of Mosquitto DLLs, OpenSSL DLLs and MQTT_Sub client. Now when I am trying to make the connection with MQTT_Pub then getting crash.

We have sorted this issue with static linking of MQTT_Pub client with old version of Mosquitto hence it is crashing. 

Thanks for you response.
